### PR TITLE
Feat(Ordering): Handle new gisaid information on sarscov2 OF 2184.5

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description
*This PR adds handling of new gisaid data on SARSCoV-2 OF 2184.5*

### How to prepare for test
- [x] ssh to clinical-db
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cgweb-stage.sh feature/#1115-cg`

### How to test
- [x] submit an 2184.5 via OF order in the order portal 

### Expected test outcome
- [x] check that order submitted without errors 
- [x] check in lims that UDF's "Region", "Region Code", "Original Lab", "Original Lab Address" and "Collection Date" were set
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [x] tests executed by PG
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Document in changelog
- [ ] Deploy this branch
- [ ] Inform to AG, MB, MR, AZ

### CODE REVIEW

File: tests/meta/orders/test_meta_orders_status.py [hunk 1]
```diff
@@ -103,18 +103,22 @@ def test_sarscov2_samples_to_status(sarscov2_order_to_submit):
 
     # THEN first sample should contain all the relevant data from the microbial order
     sample_data = data["samples"][0]
-    assert sample_data.get("priority") in "research"
-    assert sample_data["name"] == "all-fields"
     assert sample_data.get("internal_id") is None
-    assert sample_data["organism_id"] == "SARS CoV-2"
-    assert sample_data["reference_genome"] == "NC_111"
+    assert sample_data.get("priority") in "research"
```
Comment (Mropat) [inline tests/meta/orders/test_meta_orders_status.py:107]: why is this one handled differently to other values?

File: cg/models/orders/excel_sample.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
+import datetime
 from typing import List, Optional
 
+import dateutil.parser
```
Comment (moonso) [inline cg/models/orders/excel_sample.py:4]: `dateutil` has been removed from the dependencies so please don't add it again!

File: tests/meta/orders/test_meta_orders_status.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply tests/meta/orders/test_meta_orders_status.py]: There was probably a reason at some point, I'll change it to get consistency

File: cg/models/orders/excel_sample.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/models/orders/excel_sample.py:4]: good catch!

Comment (patrikgrenfeldt) [reply cg/models/orders/excel_sample.py:4]: Deadpendency should have the option to fail the build if there's suddenly more code relying on dead dependencies, now it's just failing all the time 🤕

Comment (patrikgrenfeldt) [reply cg/models/orders/excel_sample.py:4]: we still have a lot of dependencies on `dateutil` still or is my branch just containing old code? 
image

Comment (moonso) [reply cg/models/orders/excel_sample.py:4]: :( I spent a lot of time removing that before so someone must have added it again...

Comment (moonso) [reply cg/models/orders/excel_sample.py:4]: I would recommend to not use `dateutils` so that it will be easier to remove it again

Comment (patrikgrenfeldt) [reply cg/models/orders/excel_sample.py]: I removed it

**Comment by patrikgrenfeldt** [comment]
Test result: 
image
🆗 data imported without errors

**Review by Mropat — APPROVED** [review]

**Comment by patrikgrenfeldt** [comment]
Deployed to clinical-db:
image

**Pull request merged**